modules/Settings/ExportXML.php

```diff
@@ -0,0 +1,266 @@
+<?php
+/*************************************************************************************************
+* Copyright 2013 JPL TSolucio, S.L.  --  This file is a part of Attorney BackOffice.
+* You can copy, adapt and distribute the work under the "Attribution-NonCommercial-ShareAlike"
+* Vizsage Public License (the "License"). You may not use this file except in compliance with the
+* License. Roughly speaking, non-commercial users may share and modify this code, but must give credit
+* and share improvements. However, for proper details please read the full License, available at
+* http://vizsage.com/license/Vizsage-License-BY-NC-SA.html and the handy reference for understanding
+* the full license at http://vizsage.com/license/Vizsage-Deed-BY-NC-SA.html. Unless required by
+* applicable law or agreed to in writing, any software distributed under the License is distributed
+* on an  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and limitations under the
+* License terms of Creative Commons Attribution-NonCommercial-ShareAlike 3.0 (the License).
+*************************************************************************************************
+*  Module       : Export Profile Process
+*  Version      : 5.4.0
+*  Author       : JPL TSolucio, S. L.
+*************************************************************************************************/
+include_once 'vtlib/Vtiger/Module.php';
+include_once 'modules/Users/Users.php';
+require_once 'include/logging.php';
+require_once 'include/utils/UserInfoUtil.php';
+include_once 'config.inc.php';
+global $adb;
+if (!empty($_REQUEST['language'])) {
+	$lang = $_REQUEST['language'];
+} else {
+	$lang = 'en';
+}
+$category = vtlib_purify($_REQUEST['category']);
+$uri = "build/export_".$category.".xml";
```

Comment:
This is a [Path Traversal](https://owasp.org/www-community/attacks/Path_Traversal) security issue. Please fix.